cmd/wstail/rotated_logs.go

```diff
@@ -0,0 +1,63 @@
+package main
+
+import (
+	"path/filepath"
+	"sort"
+	"strconv"
+	"strings"
+)
+
+// newRotatedLogs creates a new instance of rotated from directory dir
+func newRotatedLogs(dir string, names []string) rotatedLogs {
+	var logs rotatedLogs
+	for _, name := range names {
+		baseName := filepath.Base(name)
+		if baseName == "messages.0" {
+			logs.Main = filepath.Join(dir, name)
+		}
+		if strings.HasPrefix(baseName, "messages.") && strings.HasSuffix(baseName, ".gz") {
+			logs.Compressed = append(logs.Compressed, filepath.Join(dir, name))
+		}
+	}
+	sort.Sort(naturalSortOrder(logs.Compressed))
+	return logs
+}
+
+// rotated defines a set of files managed by savelog command
+type rotatedLogs struct {
+	// Main defines a completed not yet compressed log file
+	Main string
+	// Compressed lists all compressed log files
+	Compressed []string
+}
+
+// naturalSortOrder defines a sort helper to sort filenames in
+// the natural order of their index. The filenames are assumed to
+// be of the form:
+//
+// <name>.<index>.<suffix>
+type naturalSortOrder []string
+
+func (r naturalSortOrder) Len() int      { return len(r) }
```

Comment:
`gofmt` blesses this?

I know, I know, it's dumb, but I'd rather see these declarations not compressed.